src/main/java/org/prebid/cache/metrics/GraphiteMetricsRecorder.java

```diff
@@ -18,23 +15,26 @@
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 
-import static com.codahale.metrics.MetricRegistry.name;
 
 @Slf4j
 @Component
 public class GraphiteMetricsRecorder extends MetricsRecorder
 {
+    protected static final String PREFIX_PLACEHOLDER = "prefix";
```

Comment:
For expressiveness I would recommend making prefix more discernible, for example `${prefix}` or alike.